### DISCUSSION TITLE
All users lost

### DISCUSSION THREAD

**Discussion by davbren** [babybuddy/babybuddy#310] [Support]
I have the docker implementation of the latest version and I can reach the login page but there are no users, not even admin...

**Comment by cdubz** [comment]
Hm. This happened after an update? Or a restart? What does you docker configuration look like? Curious if somehow the data was not configured as persistent.

**Comment by davbren** [comment]
just a restart, I've only just started using it. I'm pretty new to the
docker thing. Where would I find the docker configuration? Performance is
also snail pace. It's quite possible the data was not configured as
persistent.

**Comment by cdubz** [comment]
Are you using Docker Compose? If so the config we’d want to see is the `docker-compose.yml` file contents. Unfortunately I’m also not very strong with Docker so I’ll also ping on Gitter for any advise here.

**Comment by davbren** [comment]
apparently not. I used the docker-cli. It made the most sense to me. If i
need to reinstall using compose I can but ideally not...

**Comment by cdubz** [comment]
I pinged the folks on our [Gitter](https://gitter.im/babybuddy/Lobby) as there are quite a few Docker users there in case anyone has advice but I think you may be out of luck here. The primary way I have seen people use Docker for Baby Buddy is via Docker Compose. I’d recommend that and there is more info in the README about how to get things going.

**Comment by davbren** [comment]
Thanks so much for your help Christopher!

It's ok if I have to start with compose, I'm still working things out on
the server so I'll see how we go.

Thanks again,
David

**Comment by lutzky** [comment]
So, with `docker-cli`, I assume you ran the following command (from https://hub.docker.com/r/linuxserver/babybuddy):

```
docker run -d \
  --name=babybuddy \
  -e TZ=Europe/London \
  -p 8000:8000 \
  -v /path/to/appdata:/config \
  --restart unless-stopped \
  ghcr.io/linuxserver/babybuddy
```

That means your data would be saved in `/path/to/appdata`. Hopefully you didn't enter that path as-is, because it probably doesn't exist on your system. You should be able to get the configured directory using this:

```
docker inspect -f '{{json .Mounts}}' babybuddy
[{"Type":"bind","Source":"/babybuddy-docker/config","Destination":"/config","Mode":"rw","RW":true,"Propagation":"rprivate"}]
```

The `"Source"` part indicates where your actual data is stored.
